fairscale/experimental/nn/multiprocess_pipe.py

```diff
@@ -97,83 +97,70 @@ def dloss(input_rref: rpc.RRef, target_rref: rpc.RRef) -> rpc.RRef:
 
 
 class MultiProcessPipe(Module):
-    """Wraps an arbitrary :class:`nn.Sequential <torch.nn.Sequential>` module
-    to train on Pipe_. If the module requires lots of memory, Pipe will be
+    """Paritions a sequential list of modules across multiple workers to train using a
+    MultiProcessPipe_. If the module requires lots of memory, MultiProcessPipe will be
     very efficient.
     ::
 
-        model = nn.Sequential(a, b, c, d)
-        model = Pipe(model, balance=[1, 1, 1, 1], chunks=8)
+        devices = ["worker0/cuda:0", "worker1/cuda:1"]
+        layer_spec = [("linear", nn.Linear, (4, 4), {}), ("relu", nn.ReLU, (), {})]
```

Comment:
it might be good to add a comment on the expected format for the tuples within the `layer_spec` list